include/triton/Dialect/Triton/IR/Types.h

```diff
@@ -12,6 +12,8 @@ namespace mlir {
 
 unsigned getPointeeBitWidth(RankedTensorType tensorTy);
 
+bool isTilePointerType(Type type);
```

Comment:
Maybe should be `isTensorPointerType` here, since the type is `tt.ptr<tensor<>>`.

Reply:
Fixed